notebooks/post_DL3_analysis.ipynb

```diff
@@ -1,36 +1,58 @@
 {
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #10.    region_finder = WobbleRegionsFinder(n_off_regions=1)
</pre><p>I would add a comment on the usage of more off regions. Depending on the maximum value allowed for RAD_MAX_2D, Gammapy will not allow using more off regions since at low energy they will overlap. One way to overcome this problem is to restrict the value if theta angular cut up to a maximum value (e.g. 0.2 deg)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/cta-observatory/cta-lstchain/pull/1262/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='32'/>